packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -1917,14 +1917,15 @@ test('auction locks low price', async t => {
   // @ts-expect-error it's a mock
   priceAuthority.setPrice(makeRatio(0n, run.brand, baseCollateral, aeth.brand));
   await eventLoopIteration();
-  await startAuctionClock(auctioneerKit, manualTimer);
-  trace('auction started, binding lockedQuote in the vault manager state');
 
-  // Bump back up to a high price
-  // @ts-expect-error it's a mock
-  priceAuthority.setPrice(
-    makeRatio(100n * wanted, run.brand, baseCollateral, aeth.brand),
-  );
+  const schedule = await E(auctioneerKit.creatorFacet).getSchedule();
+  const priceDelay = await E(auctioneerKit.publicFacet).getPriceLockPeriod();
+  // @ts-expect-error known to be good.
```

Comment:
I forget what the error is but if it's about undefined values, better to assert so they're narrowed and we'll detect unexpected changes

Reply:
```js
  const { startTime, startDelay } = NonNullish(schedule.nextAuctionSchedule);
```